CONFIG.md

```diff
@@ -363,9 +363,8 @@ This is an example of the settings for the validate tasks:
 {
   "eslint": {
     "dir": [
-      '<%= config.srcPaths.drupal %>/**/*.js',
-      '!<%= config.srcPaths.drupal %>/**/bower/**/*.js',
-      '!<%= config.srcPaths.drupal %>/sites/**/files/**/*.js'
+      '<%= config.srcPaths.drupal %>/themes/*/js/**/*.js',
+      '<%= config.srcPaths.drupal %>/{modules,profiles}/**/*.js'
```

Comment:
I purposefully removed libraries and static from the defaults and from the docs, as it seems rare that you would want to be scanning third-party libraries (and rarer that you are writing your own in this location). I figured it would be reasonable to have these directories be opt-in, but I can add them back if that's a bad assumption.